### ISSUE TITLE
jspm bundle doesn't normalize bundled files

### ISSUE THREAD

**Issue by MrJoe** [jspm/jspm-cli#2385]
**JSPM Version:** `0.17.0-beta.47`

**Transpiler Plugin(s):** babel

**Details:**
TL;DR
When using `jspm bundle` the npm names aren't normalized causing a bundle unable to resolve it's own bundled dependencies. I am looking for the equivalent of: `builder
.bundle('src/app.js', 'outfile.js', { format: 'register', normalize: true })`

**Longer:**
I created a simple test project using `jspm init` with all the default settings.
Then I added an npm package:
`./node_modules/.bin/jspm install npm:iban`

I created an src/app.js file with the following content:
```
import * as ibanjs from "iban";

var check = ibanjs.default.isValid("123");
console.log("Is iban 123 valid?: " + check)
```

and as the final step I created an jspm bundle using `jspm bundle app`.
The resulting build.js does contain a `System.registerDynamic("npm:iban@0.0.8.json", [], ...`, `System.registerDynamic('npm:iban@0.0.8/iban.js', [], ...` and a `System.register("app/app.js", ["iban"], ...`

However the last register should actually be `System.register("app/app.js", ["npm:iban@0.0.8/iban.js"], ...` to make it work against the production system.js because packageConfigPaths can't be used in that situation.

When using systemjs-builder with `builder.bundle('src/app.js', 'outfile.js', { format: 'register', normalize: true })` I get the expected behaviour. How can I turn the normalize option on during `jspm bundle` ?
